services/power.md

```diff
@@ -6,62 +6,70 @@ A power-provider service.
 
 ## Power negotiation protocol
 
-The purpose of the power negotiation is to ensure that there is no more than ~500mA
+The purpose of the power negotiation is to ensure that there is no more than ~900mA
 delivered to the power rail.
 This is realized by limiting the number of enabled power provider services to one.
 
 Note, that it's also possible to have low-current power providers,
 which are limited to 100mA and do not run a power provider service.
 These are **not** accounted for in the power negotiation protocol.
 
-The protocol is based on `active` reports, which are always sent 
-after general device announce packets, in the same frame.
-This makes it simpler for other power services to parse them.
-
-The `active` reports contain device priority, which is formed from the maximum available current
-and remaining battery capacity.
-
-After queuing an announce with `active` report, the service enters a grace period
+The protocol is based on `shutdown` commands, which are sent as broadcast
+to the power service.
+They follow a very narrow format:
+* they need to be the only packet in the frame
+* the second word of `device_id` needs to be set to `0xAA_AA_AA_AA` (alternating 0 and 1)
+* the service index is set to `0x3d`
+* the CRC is therefore fixed
+* therefore, the packet can be recognized by reading the first 8 bytes
+
+Multi-channel power-providers (which provide a separate 900mA to each of their channels)
+will detect that a `shutdown` command is being received after 8 bytes.
+They will then briefly deactivate a switch which connects their internal and external
+Jacdac data lines.
+If the `shutdown` is still received correctly, it means it came from the internal network,
```

Comment:
When do they send this shutdown packet packet, after closing the line?

Reply:
the shutdown command causes shut down, but clients should not listen to it; added note explaining that